AI/BattleAI/BattleAI.cpp

```diff
@@ -578,6 +578,7 @@ void CBattleAI::attemptCastingSpell()
 		size_t minTurnSpan = ourUnits/3; //todo: tweak this
 
 		std::vector<battle::Units> newTurnOrder;
+
 		state.battleGetTurnOrder(newTurnOrder, amount, 2);

```

Comment:
This thing runs in parallel and invokes std::sort on a list of stacks with CMP_Stack comparator